css/custom.css

```diff
@@ -949,7 +949,36 @@ canvas {
     }
 
 /* trusties */
-
+.trusties-section {
+    background-color:#F8F9FA;
+    padding-bottom: 48px;
+}
+.trusties-img {
+    display: flex;
+    justify-content: space-around;
+    margin-top: 50px;
+}
+.trustie-avatar {
+    width: 160px;
+    height: 130px;
+}
+.trustie-avatar > img {
+    width: 160px;
+    height: 130px;
```

Comment:
Cia gali rasyti klases per kableli, nereikes kartoti to pacio stiliaus

Reply:
Nesu tikras ,bet parasius .trustie-avatar, img visiems dokumente esantiems img jis priskirs ta stiliu. Nebent butent sitam img susikurt atskira klase, kas nera blogas variantas, bet atrode tas ant to rasymo prasme.

Reply:
o jeigu parasai sitaip, nepavyksta?

.trustie-avatar,
.trustie-avatar > img {
    width: 160px;
    height: 130px;
}